app/views/lists/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1> Lists </h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% if @lists %>
+      <% @lists.each do |list| %>
```

Comment:
[suggestion] what would you think about replacing this block with a `collection_partial`?

Reply:
Yeah I think it is a great suggestion. I forgot that it existed to be honest. Thank you for reminding me!
It would definitely help cleaning up the view.